lib/routes/cjlu/yjsy/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+const host = 'https://yjsy.cjlu.edu.cn/';
+
+const titleMap = new Map([
+    ['yjstz', '中量大研究生院 —— 研究生通知'],
+    ['jstz', '中量大研究生院 —— 教师通知'],
+]);
+
+export const route: Route = {
+    path: '/yjsy/:cate',
+    categories: ['university'],
+    example: '/cjlu/yjstz',
+    parameters: {
+        cate: '订阅的类型，支持 yjstz（研究生通知）和 jstz（教师通知）',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['yjsy.cjlu.edu.cn/index/:cate/:id.htm'],
```

Comment:
`/:id` will match `.htm` in the url path as well

Reply:
Sorry for my absence during the day. I've tried patterns like `yjsy.cjlu.edu.cn/index/:cate{[a-z]+}/:id{[0-9]+}.htm` and `yjsy.cjlu.edu.cn/index/:cate{[a-z]+}.htm`, but they don't seem to work as expected.

I also referred to an existing route (lib/routes/sjtu/seiee), and here’s the relevant code:
```typescript
    radar: [
        {
            source: ['www.seiee.sjtu.edu.cn/:path.html'],
            target: '/seiee/:path',
        },
    ],
```
Could you point me in the right direction? I would greatly appreciate any suggestions!

Reply:
Additionally, there're some exapmles:
```
https://yjsy.cjlu.edu.cn/index/yjstz.htm // notifications for students page 1 of 93 (fitst page). 
https://yjsy.cjlu.edu.cn/index/yjstz/92.htm // notifications  for students page 2 of 93.
https://yjsy.cjlu.edu.cn/index/yjstz/1.htm // notifications for students page 93 of 93 (last page).

https://yjsy.cjlu.edu.cn/index/jstz.htm // notifications for teachers page 1 of 17 (first page).
https://yjsy.cjlu.edu.cn/index/jstz/16.htm // notifications for teacher page 2 of 17.
https://yjsy.cjlu.edu.cn/index/jstz/1.htm // notifications for teacher page 17 of 17 (last page).
```

Reply:
Hard-code `yjsy.cjlu.edu.cn/index/yjstz.htm` which targets `/yjsy/yjstz` and `yjsy.cjlu.edu.cn/index/jstz.htm` with `/yjsy/jstz`